app/controllers/historical_percent_changes_controller.rb

```diff
@@ -0,0 +1,29 @@
+class HistoricalPercentChangesController < ApplicationController
+  before_action :set_fixed_datum, only: %i[show update destroy]
+
+  def initialize
+   rent_charge_gateway = RentChargeGateway.new
+   fixed_data_gateway = FixedDataGateway.new
+   @view_historical_percent_changes = ViewHistoricalPercentChanges.new(
+    rent_charge_gateway: rent_charge_gateway, fixed_data_gateway: fixed_data_gateway)
```

Comment:
these initialisations can be eventually extracted to the use case factory

Reply:
I might just move them to a use case factory now, and feel the pain of the merge conflict anyway. It shouldn't be difficult to combine the two, and would neaten this up.